common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -188,3 +188,9 @@ def definition_to_xml(self, resource_fs):
         for child in self.get_children():
             self.runtime.add_block_as_child_node(child, xml_object)
         return xml_object
+
+    @property
+    def non_editable_metadata_fields(self):
+        non_editable_fields = super(SequenceDescriptor, self).non_editable_metadata_fields
+        non_editable_fields.extend([SequenceFields.is_entrance_exam, ])
```

Comment:
@mattdrayer you expect is_entrance_exam to change between course runs? I have trouble understanding that because the section wouldn't make sense to have as the first thing in a course if it is not an entrance exam.